src/accessories/indoor-unit.ts

```diff
@@ -143,18 +143,16 @@ export default class IndoorUnitAccessory {
       // Temperature of 126 from the API = null/failure
       
       if (deviceStatus.insideTemperature < 126) {
-        this.service.updateCharacteristic(this.platform.Characteristic.CurrentTemperature, deviceStatus.insideTemperature);
-      }
-      
-      else {
+	      this.service.updateCharacteristic(this.platform.Characteristic.CurrentTemperature, deviceStatus.insideTemperature);
+	      this.platform.log.info(`Indoor temperature: '${deviceStatus.insideTemperature}'`);
```

Comment:
Why log this as info? Do we have a sense of how large the log file will grow after a few months of operation on an average installation? I am aware of situations where space has been eaten up by it, eventually leading to a crash of the machine Homebridge is running on.

Reply:
Right attention! I will change it to debug.